src/plays/self-clicking-button/styles.css

```diff
@@ -0,0 +1,23 @@
+/* enter stlyes here */
+
+.self-click {
+  display: flex;
+  flex-direction: column;
```

Comment:
`column` is the default value for `flex-direction`. do you want to call out explicitly?

Reply:
Default is row.

Reply:
My bad. I meant, when `flex` is used, the default value of `flex-direction` is `column`